### ISSUE TITLE
Filter is not working when I add content to a table dynamically. 

### ISSUE THREAD

**Issue by Freid001** [javve/list.js#292]
Please scroll down to see the issue. 

===HTML===

```
<div id="driverList">
                    <input class="search" placeholder="Search" />
                    <table id="table" class="table">
                        <thead>
                            <tr>
                                <th>Name</th>
                                <th>Date</th>
                                <th>Time</th>
                            </tr>
                        </thead>
                        <tbody class="list">
                        <tr>
                           <td></td>
                           <td></td>
                           <td></td>
                       </tr>
                       </tbody>
</div>
```

===JQUERY===
The following code is runs after an AJAX request. So rows are dynamically added to the table between the tbody.

```
$('#table>tbody:last').append( '<tr>' +
                                                        '<td class="name">{NAME}</td>' +
                                                        '<td class="date">{DATE}</td>' +
                                                        '<td class="time">{TIME}<</td>' +
                                                        '</tr>');
```

So after the ajax request the table looks something like this: 

```
<div id="driverList">
                    <input class="search" placeholder="Search" />
                    <table id="table" class="table">
                        <thead>
                            <tr>
                                <th>Name</th>
                                <th>Date</th>
                                <th>Time</th>
                            </tr>
                        </thead>
                        <tbody class="list">
                        <tr>
                             <td class="name">Bob</td>'
                             <td class="date">12/10/2014</td>
                            <td class="time">11:00</td>
                       </tr>
                       <tr>
                             <td class="name">Mike</td>'
                             <td class="date">13/10/2014</td>
                            <td class="time">10:00</td>
                       </tr>
                       </tbody>
</div>
```

===THE ISSUE===
After the table has been loaded dynamically all the content is displayed correctly. But then when I try to use the search all the tables tr data disappears and nothing is found. 

So the table appears like this:

```
<div id="driverList">
                    <input class="search" placeholder="Search" />
                    <table id="table" class="table">
                        <thead>
                            <tr>
                                <th>Name</th>
                                <th>Date</th>
                                <th>Time</th>
                            </tr>
                        </thead>
                        <tbody class="list">
                       </tbody>
</div>
```

I think this issue is because I am adding data dynamically. But I need to get the data from an AJAX request :(

Any help would be gratefully appreciated.

**Comment by jmsosa77** [comment]
Hi Freid, 

I have the same problem but I can solve it. Help me please!

Thanks

**Comment by Freid001** [comment]
List.js does not support a refresh/reload function so I just created a new list object inside the AJAX request and then set that to the the var I used for the first list object. Not a very nice way of doing it really, but its a shame it does not support a refresh. I did this some time ago however, so they might have added a refresh option now. Probably best taking a look at their documentation at: http://www.listjs.com/ 

So for example:

[code]
var list= new List('list', options);

//AJAX START
list = new List('list', options); //Create a new list object as listjs does not currently support an refresh/reload feature :(
//AJAX END

[/code]
